pcsx2/Frontend/GameList.cpp

```diff
@@ -241,7 +298,7 @@ bool GameList::GetIsoListEntry(const std::string& path, GameList::Entry* entry)
 
 	return true;
 }
-
+	// clang-format on
```

Comment:
Can you surround this function properly with the clang-format on/off instead of one inside it, and also don't intent them please. so it's like

// clang-format off
void function()
{
  //blah stuff happening
}
// clang-format on

Reply:
Should be fine I think

Reply:
yep, that's fine :)